doctr/models/obj_detection/faster_rcnn/pytorch.py

```diff
@@ -31,11 +29,11 @@ def _fasterrcnn(arch: str, pretrained: bool, **kwargs: Any) -> FasterRCNN:
         "image_mean": default_cfgs[arch]['mean'],
         "image_std": default_cfgs[arch]['std'],
         "box_detections_per_img": 150,
-        "box_score_thresh": 0.15,
+        "box_score_thresh": 0.5,
```

Comment:
Are you positive this change doesn't decrease performances ?

Reply:
I experimented a bit with the model as is, and this certainly decreases the recall, but the bare predictions got me concerned on the precision side. @SiddhantBahuguna suggested having a class-specific threshold strategy and that might be the best thing to do, but for now, perhaps it might be better to maximize our precision.

Happy to revert this if you think it's best to favor recall @charlesmindee :ok_hand:

Reply:
Let's put it to 0.5 for now, but we should implement quickly a threshold by class if it leads to better results